docs/reference/search/request/highlighting.asciidoc

```diff
@@ -137,7 +137,7 @@ boundary_scanner_locale:: Controls which locale is used to search for sentence
 and word boundaries.
 
 encoder:: Indicates if the highlighted text should be HTML encoded:
```

Comment:
I just checked that for "html" not only text between highlight tags gets escaped, but the whole snippet. I think you could also change "text" here to "text snippet" or just "snippet". Would that make more sense?

Reply:
Yeah it's a bit confusing because "highlighted text" could also refer to all the text that is being highlighted (as opposed to the text between the highlighting tags).